templates/pages/checkout.html

```diff
@@ -1 +1,35 @@
-brand.html
+<!doctype html>
+<html>
+<head>
+    <base href="/">
+    <meta charset="UTF-8"/>
+    <meta name="keywords" content="checkout"/>
+    <meta name="description" content="Bigcommerce checkout"/>
+    <meta name="robots" content="noindex, nofollow">
+    <meta name="viewport" content="width=device-width">
+    <link rel="SHORTCUT ICON" href="{{ FaviconUrl }}" />
+    {{{ open_sans_stylesheet }}}
+    {{{ ng_checkout_stylesheet }}}
+    {{{ stylesheet '/assets/css/optimized-checkout.css' }}}
+    {{ getFontsCollection }}
```

Comment:
@snaderiBC : looks fine to me, loading after the main stylesheets similar to https://github.com/bigcommerce/stencil/blob/master/templates/layout/base.html#L11

I guess it could also come after `{{ OptimizedCheckoutWebdavCss }}` 🍹